[2025-07-21 10:14:03] Exception in maybe_run_webcmd:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 3151, in maybe_run_webcmd
    cmd = json.load(f)
          ^^^^^^^^^^^^
  File "/usr/lib/python3.11/json/__init__.py", line 293, in load
    return loads(fp.read(),
           ^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

[2025-07-21 10:06:43] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 1010, in process_command
    handle_join_series(bases, suffixes, overall_m)
    ^^^^^^^^^^^^^^^^^^
NameError: name 'handle_join_series' is not defined

[2025-07-21 10:06:08] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 1010, in process_command
    handle_join_series(bases, suffixes, overall_m)
    ^^^^^^^^^^^^^^^^^^
NameError: name 'handle_join_series' is not defined

[2025-07-21 10:06:05] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 1010, in process_command
    handle_join_series(bases, suffixes, overall_m)
    ^^^^^^^^^^^^^^^^^^
NameError: name 'handle_join_series' is not defined

[2025-07-21 10:06:02] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 1010, in process_command
    handle_join_series(bases, suffixes, overall_m)
    ^^^^^^^^^^^^^^^^^^
NameError: name 'handle_join_series' is not defined

[2025-07-21 09:12:25] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 1005, in process_command
    handle_join_series(bases, suffixes, overall_m)
  File "/home/drx/DRX/drx_main.py", line 863, in handle_join_series
    process_command(cmd, defer_message_timer_update=True)  # do NOT allow timer update during the segment
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: process_command() got an unexpected keyword argument 'defer_message_timer_update'

[2025-07-21 02:24:47] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 982, in process_command
    break
NameError: name 'play_track_by_number' is not defined

[2025-07-21 02:24:33] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 982, in process_command
    played = play_track_by_number(digits, suffixes)
             ^^^^^^^^^^^^^^^^^^^^
NameError: name 'play_track_by_number' is not defined

[2025-07-21 02:23:57] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 984, in process_command
    played = play_track_by_number(digits, suffixes)
             ^^^^^^^^^^^^^^^^^^^^
NameError: name 'play_track_by_number' is not defined

[2025-07-21 02:23:52] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 984, in process_command
    played = play_track_by_number(digits, suffixes)
             ^^^^^^^^^^^^^^^^^^^^
NameError: name 'play_track_by_number' is not defined

[2025-07-21 02:19:23] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 982, in process_command
    played = play_track_by_number(digits, suffixes)
             ^^^^^^^^^^^^^^^^^^^^
NameError: name 'play_track_by_number' is not defined

[2025-07-21 02:18:36] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 982, in process_command
    played = play_track_by_number(digits, suffixes)
             ^^^^^^^^^^^^^^^^^^^^
NameError: name 'play_track_by_number' is not defined

[2025-07-21 02:18:26] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 982, in process_command
    played = play_track_by_number(digits, suffixes)
             ^^^^^^^^^^^^^^^^^^^^
NameError: name 'play_track_by_number' is not defined

[2025-07-21 02:15:10] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 982, in process_command
    played = play_track_by_number(digits, suffixes)
             ^^^^^^^^^^^^^^^^^^^^
NameError: name 'play_track_by_number' is not defined

[2025-07-21 02:12:21] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 1001, in process_command
    played = play_track_by_number(digits, suffixes)
             ^^^^^^^^^^^^^^^^^^^^
NameError: name 'play_track_by_number' is not defined

[2025-07-21 02:09:53] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 1001, in process_command
    played = play_track_by_number(digits, suffixes)
             ^^^^^^^^^^^^^^^^^^^^
NameError: name 'play_track_by_number' is not defined

[2025-07-21 02:09:36] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 1001, in process_command
    played = play_track_by_number(digits, suffixes)
             ^^^^^^^^^^^^^^^^^^^^
NameError: name 'play_track_by_number' is not defined

[2025-07-21 02:09:32] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 1001, in process_command
    played = play_track_by_number(digits, suffixes)
             ^^^^^^^^^^^^^^^^^^^^
NameError: name 'play_track_by_number' is not defined

[2025-07-21 02:09:23] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 1001, in process_command
    played = play_track_by_number(digits, suffixes)
             ^^^^^^^^^^^^^^^^^^^^
NameError: name 'play_track_by_number' is not defined

[2025-07-21 02:09:18] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 1001, in process_command
    played = play_track_by_number(digits, suffixes)
             ^^^^^^^^^^^^^^^^^^^^
NameError: name 'play_track_by_number' is not defined

[2025-07-21 02:09:13] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 1001, in process_command
    played = play_track_by_number(digits, suffixes)
             ^^^^^^^^^^^^^^^^^^^^
NameError: name 'play_track_by_number' is not defined

[2025-07-21 01:14:13] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 1033, in process_command
    wavs = get_minimal_alert_wavs()
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/drx/DRX/drx_main.py", line 973, in get_minimal_alert_wavs
    return get_configured_minimal_alert_wavs()  # or your actual function
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
NameError: name 'get_configured_minimal_alert_wavs' is not defined

[2025-07-21 01:14:10] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 1033, in process_command
    wavs = get_minimal_alert_wavs()
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/drx/DRX/drx_main.py", line 973, in get_minimal_alert_wavs
    return get_configured_minimal_alert_wavs()  # or your actual function
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
NameError: name 'get_configured_minimal_alert_wavs' is not defined

[2025-07-21 01:13:23] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 1038, in process_command
    alert_played = play_minimal_alert_with_cos_interrupt()
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/drx/DRX/drx_main.py", line 974, in play_minimal_alert_with_cos_interrupt
    wavs = get_minimal_alert_wavs()
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/drx/DRX/drx_main.py", line 971, in get_minimal_alert_wavs
    return get_configured_minimal_alert_wavs()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
NameError: name 'get_configured_minimal_alert_wavs' is not defined

[2025-07-21 01:13:19] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 1038, in process_command
    alert_played = play_minimal_alert_with_cos_interrupt()
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/drx/DRX/drx_main.py", line 974, in play_minimal_alert_with_cos_interrupt
    wavs = get_minimal_alert_wavs()
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/drx/DRX/drx_main.py", line 971, in get_minimal_alert_wavs
    return get_configured_minimal_alert_wavs()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
NameError: name 'get_configured_minimal_alert_wavs' is not defined

[2025-07-21 00:58:21] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 1005, in process_command
    played_ok = play_minimal_alert_with_cos_interrupt()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/drx/DRX/drx_main.py", line 977, in play_minimal_alert_with_cos_interrupt
    wavs = get_minimal_alert_wavs()
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/drx/DRX/drx_main.py", line 973, in get_minimal_alert_wavs
    return get_configured_minimal_alert_wavs()  # or your actual function
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
NameError: name 'get_configured_minimal_alert_wavs' is not defined

[2025-07-21 00:58:19] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 1005, in process_command
    played_ok = play_minimal_alert_with_cos_interrupt()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/drx/DRX/drx_main.py", line 977, in play_minimal_alert_with_cos_interrupt
    wavs = get_minimal_alert_wavs()
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/drx/DRX/drx_main.py", line 973, in get_minimal_alert_wavs
    return get_configured_minimal_alert_wavs()  # or your actual function
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
NameError: name 'get_configured_minimal_alert_wavs' is not defined

[2025-07-21 00:57:23] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 1005, in process_command
    played_ok = play_minimal_alert_with_cos_interrupt()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/drx/DRX/drx_main.py", line 977, in play_minimal_alert_with_cos_interrupt
    wavs = get_minimal_alert_wavs()
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/drx/DRX/drx_main.py", line 973, in get_minimal_alert_wavs
    return get_configured_minimal_alert_wavs()  # or your actual function
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
NameError: name 'get_configured_minimal_alert_wavs' is not defined

[2025-07-21 00:57:12] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 1005, in process_command
    played_ok = play_minimal_alert_with_cos_interrupt()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/drx/DRX/drx_main.py", line 977, in play_minimal_alert_with_cos_interrupt
    wavs = get_minimal_alert_wavs()
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/drx/DRX/drx_main.py", line 973, in get_minimal_alert_wavs
    return get_configured_minimal_alert_wavs()  # or your actual function
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
NameError: name 'get_configured_minimal_alert_wavs' is not defined

[2025-07-21 00:56:57] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 1005, in process_command
    played_ok = play_minimal_alert_with_cos_interrupt()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/drx/DRX/drx_main.py", line 977, in play_minimal_alert_with_cos_interrupt
    wavs = get_minimal_alert_wavs()
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/drx/DRX/drx_main.py", line 973, in get_minimal_alert_wavs
    return get_configured_minimal_alert_wavs()  # or your actual function
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
NameError: name 'get_configured_minimal_alert_wavs' is not defined

[2025-07-21 00:28:14] Exception in maybe_run_webcmd:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 3199, in maybe_run_webcmd
    cmd = json.load(f)
          ^^^^^^^^^^^^
  File "/usr/lib/python3.11/json/__init__.py", line 293, in load
    return loads(fp.read(),
           ^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

[2025-07-20 23:47:28] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 4537, in process_command
    speak_wx_alerts_minimal(keep_busy=False)
    ^^^^^^^^^^^^^^^^^^^^^^^
NameError: name 'speak_wx_alerts_minimal' is not defined

[2025-07-20 23:47:23] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 4537, in process_command
    speak_wx_alerts_minimal(keep_busy=False)
    ^^^^^^^^^^^^^^^^^^^^^^^
NameError: name 'speak_wx_alerts_minimal' is not defined

[2025-07-20 23:47:14] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 4508, in process_command
    speak_wx_alerts_minimal(keep_busy=bool(digits))
    ^^^^^^^^^^^^^^^^^^^^^^^
NameError: name 'speak_wx_alerts_minimal' is not defined

[2025-07-20 23:47:06] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 4508, in process_command
    speak_wx_alerts_minimal(keep_busy=bool(digits))
    ^^^^^^^^^^^^^^^^^^^^^^^
NameError: name 'speak_wx_alerts_minimal' is not defined

[2025-07-20 23:47:04] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 4508, in process_command
    speak_wx_alerts_minimal(keep_busy=bool(digits))
    ^^^^^^^^^^^^^^^^^^^^^^^
NameError: name 'speak_wx_alerts_minimal' is not defined

[2025-07-20 23:01:50] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 1008, in process_command
    segments, overall_m, is_join = parse_join_series(command.strip())
                                   ^^^^^^^^^^^^^^^^^
NameError: name 'parse_join_series' is not defined

[2025-07-20 22:59:56] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 1008, in process_command
    segments, overall_m, is_join = parse_join_series(command.strip())
                                   ^^^^^^^^^^^^^^^^^
NameError: name 'parse_join_series' is not defined

[2025-07-20 22:52:38] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 1008, in process_command
    segments, overall_m, is_join = parse_join_series(command.strip())
                                   ^^^^^^^^^^^^^^^^^
NameError: name 'parse_join_series' is not defined

[2025-07-20 22:52:27] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 1008, in process_command
    segments, overall_m, is_join = parse_join_series(command.strip())
                                   ^^^^^^^^^^^^^^^^^
NameError: name 'parse_join_series' is not defined

[2025-07-20 22:52:21] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 1008, in process_command
    segments, overall_m, is_join = parse_join_series(command.strip())
                                   ^^^^^^^^^^^^^^^^^
NameError: name 'parse_join_series' is not defined

[2025-07-20 22:52:20] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 1008, in process_command
    segments, overall_m, is_join = parse_join_series(command.strip())
                                   ^^^^^^^^^^^^^^^^^
NameError: name 'parse_join_series' is not defined

[2025-07-20 22:52:07] Exception in process_command:
Traceback (most recent call last):
  File "/home/drx/DRX/drx_main.py", line 1008, in process_command
    segments, overall_m, is_join = parse_join_series(command.strip())
                                   ^^^^^^^^^^^^^^^^^
NameError: name 'parse_join_series' is not defined

